src/transformers/modeling_tf_utils.py

```diff
@@ -486,16 +486,17 @@ def _get_resized_embeddings(self, old_embeddings, new_num_tokens=None) -> tf.Var
         # todo: initializer range is not always passed in config.
         init_range = getattr(self.config, "initializer_range", 0.02)
         new_embeddings = self.add_weight(
-            "weight",
+            name=word_embeddings.name.split(":")[0],
```

Comment:
Fix a naming issue. When the resized is created, the name is changed as well and raises a naming issue when the updated model saved and then loaded.
```
from transformers import BertConfig, TFBertForMaskedLM
model = TFBertForMaskedLM(BertConfig())
model(model.dummy_inputs)
model.resize_token_embeddings(28996)
model.save_pretrained("here")
model = TFBertForMaskedLM.from_pretrained("here")
```
Gives
```
Some layers from the model checkpoint at here were not used when initializing TFBertForMaskedLM: ['']
- This IS expected if you are initializing TFBertForMaskedLM from the checkpoint of a model trained on another task or with another architecture (e.g. initializing a BertForSequenceClassification model from a BertForPreTraining model).
.....
```

Weights and bias are not properly loaded. After naming fix:
```
All model checkpoint layers were used when initializing TFBertForMaskedLM.

All the layers of TFBertForMaskedLM were initialized from the model checkpoint at here.
If your task is similar to the task the model of the checkpoint was trained on, you can already use TFBertForMaskedLM for predictions without further training.
```